### ISSUE TITLE
Submissions hijacked by captchas don't allow for validateCustom events

### ISSUE THREAD

**Issue by jedso** [verbb/formie#2170]
### Describe the bug

In my form, I rely on some custom validation logic before the form is finally submitted with the `onFormieCustomValidate` event. This works fine, but when a captcha integration is enabled (reCAPTCHA v3 in my case), the calls to `this.validatePayment()` and `this.validateCustom()` never run in `processSubmit`:
https://github.com/verbb/formie/blob/1056fc8825276a8498190925be793add625a040f/src/web/assets/frontend/src/js/formie-form-base.js#L105-L131

For `this.validatePayment()`, this isn't a concern because once we finally get to re-submission in `onVerify`, there's logic to call it:
https://github.com/verbb/formie/blob/1056fc8825276a8498190925be793add625a040f/src/web/assets/frontend/src/js/captchas/recaptcha-v3.js#L166-L174

...but there's no such logic for calling `validateCustom()`. So I think the simple solution here is just to add `validateCustom()` for each of the built-in captcha integrations:
```js
    if (this.submitHandler.validatePayment() && this.submitHandler.validateCustom()) {
        this.submitHandler.submitForm();
    }
```

### Steps to reproduce

1. Create a form without a captcha integration enabled, and include custom JS that runs logic when the `onFormieCustomValidate` event is dispatched. E.g. a simple console.log()
2. Observe how the log message appears on submission
3. Enable a captcha integration on the form (reCAPTCHA v3 in my case)
4. Observe how the `onFormieCustomValidate` logic will never run on form submission with the captcha enabled

### Form settings

- Multi-page form: Yes
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

### Craft CMS version

5.5.2

### Plugin version

3.0.14

### Multi-site?

No

**Comment by engram-design** [comment]
Yep, that's a good point. I don't _love_ the way we currently handle this with captchas, but it'll do for the time being until I can restructure things to be promises (a breaking change).

Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-5 as 3.0.14"`.

**Comment by engram-design** [comment]
Fixed in [3.0.15](https://github.com/verbb/formie/releases/tag/3.0.15)
